### DISCUSSION TITLE
关于使用代替bot时查询、签到功能出现错误的问题

### DISCUSSION THREAD

**Discussion by CeciliaAki** [Arondight/Adachi-BOT#792] [Q&A] [answered]
大佬们好，我按照教程将UM开头的cookie复制到了config当中的cookie.yml文件中，但是在实际使用bot的时候，无法查询自身账号信息，也无法开启自动签到功能，请问这种情况可能是哪个环节出了问题呢？

![失败](https://user-images.githubusercontent.com/103917391/163775886-096a939c-b5d3-40a0-889c-400a05b5bfbe.png)

**Comment by Arondight** [comment]
1. 查不到数据，检查是否存在 uid，或者看看 #371 
2. 我不知道，你需要问作者……

@ixCiel 要不要在你的 fork 中开启 issue 或者 discussion ？这样人民群众用你的项目时，有问题可以去你那边问了。

**Reply by CeciliaAki** [reply]
你好！
uid检查后存在，也开启了个人主页卡片和角色数据可见，可以直接使用“uid <>”，“深渊 <>”，“上期深渊  <>”功能，但不能使用米游社、查询单个角色、签到札记便笺等功能。
谢谢你的回复！我会再等待其他答案^^

**Reply by ixCiel** [reply]
那么问题来了，绑定需要的是米游社通行证id，不是游戏的uid。Orz

**Reply by CeciliaAki** [reply]
两位大佬们好！我犯了一个愚蠢的错误即将米游社id错认为游戏uid，经过指正后绑定成功。
我尝试设置私人cookie时发现新的教程，即用火狐浏览器的教程，在此之前我使用的是打开网页控制台获取cookie的方法
（所使用的浏览器为edge），于是我尝试更改方法，更改方法以后我成功设置了私人cookie，大部分功能都可以正常使用！
唯有一点小问题是在发送“米游社签到 <uid>”时机器人不做反应，但开启自动签到功能时显示已成功开启自动签到，只是回到游戏中检查发现并未收到邮件，不知是否存在时间差问题，我会耐心等待，或探索其他自动签到的方法。
感谢两位的耐心指正！

**Reply by CeciliaAki** [reply]
后续：签到成功，收到邮件。

**Reply by ixCiel** [reply]
我的问题，command.yml里面没修改。具体命令可以看command.yml。note里面就是我增加的。因为一开始只打算弄树脂的。后来越加越多Orz，但是我又比较懒，就弄一起了。

**Comment by ixCiel** [comment, marked as answer]
感觉像是米游社那边没有公开数据
米游社app->我的->个人主页->管理
把个人主页卡片和角色详情数据公开
另外，签到等功能要设置私人cookie

**Reply by CeciliaAki** [reply]
作者你好！我检查过后米游社已公开数据。
在设置cookie时，我将网页上的cookie从UM开始复制，向机器人发送“设置Cookie <我复制的cookie>”后，机器人回复获取游戏UID失败：[object Object]。这是否算作设置私人cookie失败？

**Reply by ixCiel** [reply]
看上面的回复，绑定需要的是米游社的Id，而不是游戏的uid
